utils/build-script-impl

```diff
@@ -1471,6 +1471,7 @@ for host in "${ALL_HOSTS[@]}"; do
                 runtimes=()
                 [[ ! ${SKIP_BUILD_LIBCXX} || ( ! ${SKIP_BUILD_COMPILER_RT} && $(uname -s) == Darwin ) ]] && {
                   runtimes+=(libcxxabi libcxx)
+                  cmake_options+=(-DLLVM_BUILD_RUNTIMES=NO)
```

Comment:
Doesn't this prevent compiler-rt from being built?

Reply:
No, it just excludes libc++ from the `all` target.